packages/compass/src/main/application.ts

```diff
@@ -145,6 +145,16 @@ class CompassApplication {
         },
         authPortalUrl: 'https://account-dev.mongodb.com/account/login',
       },
+      'atlas-local': {
+        atlasApiBaseUrl: 'http://localhost:8080',
+        atlasApiUnauthBaseUrl:
+          'https://cloud-dev.mongodb.com/api/private/unauth',
```

Comment:
Should this also be local host?
```suggestion
          'http://localhost:8080/api/private/unauth',
```
I can give it a manual test on local if it helps to confirm.

Reply:
that would be awesome!